### DISCUSSION TITLE
Onnx Web YOLO-Seg <- how to interpret the outputs

### DISCUSSION THREAD

**Discussion by Piebee007** [onnx/onnx#6178] [Q&A]
Hiya,
So I have recently made an ONNX web application for object detection and now moving to segmentation.
I have followed the crack-seg on the ultralytics website and have converted it to an ONNX model (with an input shape of 320*320).
I am now just stuck trying to interpret the output data. 
Below is the picture of the output. 
I understand the first 10500 numbers on the output0 (the bounding box x, y, width and height then the confidence of the one class)
I now have no clue how to understand what output1 is, i imagine it is 32 lots of 80 x and y positions but just wonder if any one else knows what it could be.
Thanks!
![Capture](https://github.com/onnx/onnx/assets/74003575/105930d7-715e-4052-aafe-e8f2a4669ff2)

**Comment by MCtop4ik** [comment]
I myself did not fully understand, but the explanation of output1 was in this [article](https://dev.to/andreygermanov/how-to-implement-instance-segmentation-using-yolov8-neural-network-3if9)
